README.md

```diff
@@ -80,14 +81,18 @@ if you want to disable it use selinux=0 on the kernel cmdline
 4. Run virtual machine with the image
 -------------------------------------
 ```
-With SELinux
-runqemu ledge-qemux86-64 nographic qemuparams="-m 4096"
+Navigate to build image directyry:
+cd ./build-rpb/tmp-rpb-glibc/deploy/images/ledge-qemuarm64
+
+Then run:
+runqemu ledge-qemuarm64 wic
 
-Without SELinux
-runqemu ledge-qemux86-64 nographic qemuparams="-m 4096" bootparams="selinux=0"
 ```
+(you should see that tfa, optee, uboot, kernel then apps boots, and you will be able to login inside vm.)
 
-(you should see that kernel then apps boots, and you will be able to login inside vm.)
+Note: on first boot selinux will issue reboot command but correct qemu for aarch64 does not support reboot
+in secure mode. Qemu virtual machine will shutdown and hang. So you need to kill qemu process and rerun qemu
+command. Second boot will allow you to boot into the system.
 
```

Comment:
On first boot SElinux re-labeling will force a reboot which currently hangs using TF-A.
Restart the QEMU process manually and the image will run properly